docs/meshstack.aks-platform-quickstart-guide.md

```diff
@@ -0,0 +1,223 @@
+---
+id: meshstack.aks-platform-quickstart-guide
+title: AKS Platform Quickstart Guide
+---
+
+
+# Introduction
+
+Welcome to the **First Platform: AKS guide**! This guide provides step-by-step instructions to help you quickly set up
+your first meshStack platform integration with Azure Kubernetes Service (AKS).
+
+# Goals
+
+By the end of this guide, you will have achieved:
+
+1. Creating a platform integration with AKS. This will allow:
+   - provisioning new namespaces
+   - assign users
+   - assign tags (optional)
+2. Successfully testing the AKS integration
+
+This allows your internal users to open up meshStack and provision themselves namespaces, including identity & access management.
```

Comment:
```suggestion
This allows your users to open up meshStack and provision themselves namespaces and authorization on AKS.
```
You could reference application teams to make it clearer. The internal user should be clear from context of the docs